flink/v1.14/flink/src/test/java/org/apache/iceberg/flink/source/enumerator/TestContinuousSplitPlannerImpl.java

```diff
@@ -396,7 +396,7 @@ public void testIncrementalFromSnapshotTimestampWithEmptyTable() throws Exceptio
 
     AssertHelpers.assertThrows("Should detect invalid starting snapshot timestamp",
         IllegalArgumentException.class,
-        "Cannot find a snapshot older than 1970-01-01 00:00:00.001",
+        "Cannot find a snapshot older than 1970-01-01T00:00:00.001+00:00",
```

Comment:
Do we want to add the `T`? Seems like that makes it harder to read. Since this string is primarily for people, making it easy to read seems worth not conforming to ISO-8601 exactly. Maybe we should add a space before the zone offset, too?

Reply:
It feels more natural to adhere to the ISO standard as that is pretty well understood. Also people could really just copy/paste this (not sure if anyone would actually do that) and parse it. I don't see any clear benefit to not adhere to the ISO standard, but happy to change if others think we should use something non-standard here.

Reply:
+1 to ISO 8601.

Also the `T` avoids a space, which IMHO makes copy-paste and automated processing easier.

Reply:
Seems reasonable to me.